test/sql/test_storage_volume/R/set_as_default

```diff
@@ -23,6 +19,10 @@ ALTER STORAGE VOLUME storage_volume_default_1 SET ("aws.s3.region"="us-west-1",
 SET storage_volume_default_1 AS DEFAULT STORAGE VOLUME;
 -- result:
 -- !result
+DESC STORAGE VOLUME 'storage_volume_default_1';
```

Comment:
What is the reason for the change?

Reply:
case is not stable, STORAGE VOLUME must has one default value, have discussed with case owner